components/typography.css

```diff
@@ -1 +1,85 @@
-@import url(/stylesheets/main.css);;
\ No newline at end of file
+@import url(/stylesheets/main.css);
+@import url("https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/an-old-hope.min.css");
+
+/* font-sizes */
+.font-xxl {
+  font-size: 48px;
```

Comment:
why not rems?

Reply:
https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css
there's a war going on but each has its own cons and pros

Reply:
no particular reason you want rems to make text responsive and but as of now I am not optimizing for that.